src/Symfony/Bundle/FrameworkBundle/Resources/config/services.php

```diff
@@ -0,0 +1,216 @@
+            ->alias(ParameterBagInterface::class, 'parameter_bag')
+
+        ->set('event_dispatcher', EventDispatcher::class)
+            ->public()
+            ->tag('container.hot_path')
+            ->alias(EventDispatcherInterfaceComponentAlias::class, 'event_dispatcher')
+            ->alias(EventDispatcherInterface::class, 'event_dispatcher')
+
+        ->set('http_kernel', HttpKernel::class)
+            ->public()
+            ->arg('$resolver', service('controller_resolver'))
+            ->arg('$argumentResolver', service('argument_resolver'))
+            ->tag('container.hot_path')
+            ->alias(HttpKernelInterface::class, 'http_kernel')
+
+        ->set('request_stack', RequestStack::class)
+            ->public()
+            ->alias(RequestStack::class, 'request_stack')
+
+        ->set('url_helper', UrlHelper::class)
+            ->args([
+                service(RequestStack::class),
+                service('router.request_context')->ignoreOnInvalid(),
+            ])
+            ->alias(UrlHelper::class, 'url_helper')
+
+        ->set('cache_warmer', CacheWarmerAggregate::class)
+            ->public()
+            ->arg('$warmers', tagged_iterator('kernel.cache_warmer'))
+            ->tag('container.no_preload')
+
+        ->set('cache_clearer', ChainCacheClearer::class)
+            ->public()
+            ->arg('$clearers', tagged_iterator('kernel.cache_clearer'))
+
+        ->set('reverse_container', ReverseContainer::class)
+            ->args([
+                service('service_container'),
+                service_locator([]),
+            ])
+            ->alias(ReverseContainer::class, 'reverse_container')
+
+        ->set('container.env_var_processor', EnvVarProcessor::class)
+            ->tag('container.env_var_processor')
+            ->args([
+                service('service_container'),
+                tagged_iterator('container.env_var_loader'),
+            ])
+
+        ->set('container.getenv', Closure::class)
+            ->factory([Closure::class, 'fromCallable'])
+            ->args([
+                [service('service_container'), 'getEnv'],
+            ])
+            ->tag('routing.expression_language_function', ['function' => 'env'])
+
+        ->set('container.env', LazyString::class)->abstract() // inherit from this service to lazily access env vars
+            ->factory([LazyString::class, 'fromCallable'])
+            ->args([
+                service('container.getenv'),
+            ])
+
+        ->set('kernel')
+            ->synthetic()
+            ->public()
+            ->alias(KernelInterface::class, 'kernel')
+
+        ->set('filesystem', Filesystem::class)
+            ->public()
+            ->alias(Filesystem::class, 'filesystem')
+
+        ->set('file_locator', FileLocator::class)
+            ->alias(FileLocator::class, 'file_locator')
+
+        ->set('uri_signer', UriSigner::class)
+            ->args([
+                param('%kernel.secret%'),
```

Comment:
param('kernel.secret')

Reply:
Thank you. Fixed in eb0f8ad7296f5bc506c4ed7d55793dac017dfedb